File: GTTS/INL.TrainingService.Models/ITrainingEventInstructor_Item.cs
using System;

namespace INL.TrainingService.Models
{
    public interface ITrainingEventInstructor_Item
    {
        long PersonID { get; set; }
        long TrainingEventID { get; set; }
        long? PersonsVettingID { get; set; }
        bool IsTraveling { get; set; }
        int? DepartureCityID { get; set; }
        DateTime? DepartureDate { get; set; }
        DateTime? ReturnDate { get; set; }
        int? VisaStatusID { get; set; }
        int? PaperworkStatusID { get; set; }
        int? TravelDocumentStatusID { get; set; }
        bool RemovedFromEvent { get; set; }
        int? RemovalReasonID { get; set; }
        int? RemovalCauseID { get; set; }
        DateTime? DateCanceled { get; set; }
        string Comments { get; set; }
        int? ModifiedByAppUserID { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/itraining-event-instructor_item.ts




export interface ITrainingEventInstructor_Item {
  
	PersonID: number;
	TrainingEventID: number;
	PersonsVettingID?: number;
	IsTraveling: boolean;
	DepartureCityID?: number;
	DepartureDate?: Date;
	ReturnDate?: Date;
	VisaStatusID?: number;
	PaperworkStatusID?: number;
	TravelDocumentStatusID?: number;
	RemovedFromEvent: boolean;
	RemovalReasonID?: number;
	RemovalCauseID?: number;
	DateCanceled?: Date;
	Comments: string;
	ModifiedByAppUserID?: number;
}





